src/Microsoft.DotNet.ProjectJsonMigration/Rules/MigratePackageDependenciesAndToolsRule.cs

```diff
@@ -59,8 +59,11 @@ public void Apply(MigrationSettings migrationSettings, MigrationRuleInputs migra
             foreach (var targetFramework in targetFrameworks)
             {
                 MigrationTrace.Instance.WriteLine(String.Format(LocalizableStrings.MigratingFramework, targetFramework.FrameworkName.GetShortFolderName()));
-                
-                MigrateImports(migrationRuleInputs.CommonPropertyGroup, targetFramework);
+
+                MigrateImports(
+                    migrationRuleInputs.CommonPropertyGroup,
+                    targetFramework,
+                    migrationRuleInputs.ProjectContexts.Count() > 1);
```

Comment:
For sure, I actually had a mental note to do that and forgot. Thanks for pointing it out.